src/components/Chat/InputMessageV2.tsx

```diff
@@ -64,8 +64,8 @@ const styles = StyleSheet.create({
   },
   input: {
     flex: 1,
-    backgroundColor: COLORS.lightgrey,
-    color: COLORS.black,
+    backgroundColor: COLORS.gray100,
+    color: COLORS.white2,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes in the `input` style are significant. The background color has been changed from light grey to gray100, and the text color has been changed from black to white2. Ensure that these changes align with the overall design and that they provide sufficient contrast for readability.

```diff
-    backgroundColor: COLORS.lightgrey,
-    color: COLORS.black,
+    backgroundColor: COLORS.gray100,
+    color: COLORS.white2,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->